src/State/HubInterface.php

```diff
@@ -23,7 +23,7 @@
  * stack of pairs of clients and scopes. It is the main entry point to talk
  * with the Sentry client.
  *
- * @method string|null captureCheckIn(string $slug, CheckInStatus $status, $duration = null, ?MonitorConfig $upsertMonitorConfig = null, ?string $checkInId = null) Captures a check-in
+ * @method string|null captureCheckIn(string $slug, CheckInStatus $status, int|float|null $duration = null, ?MonitorConfig $monitorConfig = null, ?string $checkInId = null) Captures a check-in
```

Comment:
Does this make phpstan happier now? Maybe we can update the baseline file.

Reply:
The baseline hasn't changed since 2 months, why should this change affect it?

Reply:
Ahh, I meant we might be able to update the doc blocks to `{@inheritdoc}`.

Reply:
It doesn't work, but we can use `{@inheritdoc}` and just add `@param int|float|null $duration` next to it to make PHPStan happier, which is better than the current situation anyway